javalib/src/main/scala/java/util/regex/Pattern.scala

```diff
@@ -0,0 +1,109 @@
+package java.util.regex
+
+import scalanative.native._, stdlib._, stdio._
+import scalanative.runtime.struct
+
+@extern
+object Regex {
```

Comment:
It should also go to `scala.scalanative.posix` package, as that's a binding for a posix API.